schemas/com.snowplowanalytics.snowplow.ecommerce/cart/jsonschema/1-0-0

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a cart entity in Ecommerce",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+        "name": "cart",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "cart_id": {
+            "description": "The unique ID representing this cart.",
+            "type": ["string","null"],
+            "maxLength": 4096
+        },
+        "total_value": {
```

Comment:
Is this also a monetary field?  If yes, I think it would be great to put `multipleOf: 0.01` like you've done on other fields.  If you do this, then it will get stored in the warehouse as a decimal type, and that is the standard way to store money fields in databases.

Reply:
Good catch, done!